### PR TITLE
chore(deps): update dependencies

### PR DESCRIPTION
**Pull request by singingwolfboy**

### CODE REVIEW

File: __tests__/helpers.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export function makeActionSpies(shadow = false): ActionSpies {
 }
 
 function makePgClientMock() {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
```
Comment (benjie) [inline __tests__/helpers.ts]: Please just disable this lint rule globally.

Comment (singingwolfboy) [reply __tests__/helpers.ts]: Fixed!

File: package.json [hunk 2]
```diff
@@ -37,29 +37,27 @@
...
-    "@types/chalk": "^2.2.0",
-    "@types/chokidar": "^2.1.3",
-    "@types/pg": "^7.11.1",
-    "chalk": "^2.4.2",
-    "chokidar": "^3.1.1",
-    "pg": ">=6.5 <8",
+    "@types/pg": "^7.14.1",
+    "chalk": "^3.0.0",
+    "chokidar": "^3.3.1",
+    "pg": "^7.18.1",
```
Comment (singingwolfboy) [inline package.json]: Looks like `yarn upgrade --latest` converted `>=6.5 <8` to `^7.18.1`. Should I keep this change, or revert it?

Comment (benjie) [reply package.json]: Please revert it 👍

Comment (singingwolfboy) [reply package.json]: Done

**Pull request merged**